### DISCUSSION TITLE
Is there any way to disable all themes?

### DISCUSSION THREAD

**Discussion by MylesWritesCode** [LazyVim/LazyVim#648] [Q&A] [answered]
I've added the following code to disable the theme in vscode, but colors seem to still be passing through the neovim plugin. I can't find where the other theme is; is there a way to completely disable themes while in vscode?

```
if vim.g.vscode then
  return {
    { "folke/tokyonight.nvim", enabled = false },
  }
else
  return {}
end
```

**Comment by odysseyalive** [comment]
Haven't tested it, but I believe what you're looking for is explained in the comments of the file you have in .confit/nvim/lua/config/lazy.lua. At the bottom of that file is a disabled_plugins feature. I'm pretty sure you'll want to add colorscheme to that list. I'm not sure if the install line above it will also need to be commented. I think it's worth playing with.

**Reply by MylesWritesCode** [reply]
Yeah I've tried something like that, it doesn't seem to work. Thanks for your reply though!

```
disabled_plugins = {
  ...
  "colorscheme",                         // tried first
  "lazyvim.plugins.colorscheme"  // also doesn't seem to work
}
```

**Comment by folke** [comment, marked as answer]
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      -- don't let LazyVim load a colorscheme
      colorscheme = function() end
    }
  }
}
```

**Reply by MylesWritesCode** [reply]
Oh wow thanks, that worked perfectly 👍
